service/pkg/codes/codes.go

```diff
@@ -0,0 +1,36 @@
+package codes
+
+type Code int
+
+const (
+	Unknown Code = iota
+
+	JSONEncodeError
+
+	BCCGetFromTableError
+	BCCSetToTableError
+	BCCNilValueFromTableError
+)
+
+var statusMessages = map[Code]string{
+	Unknown: "Unknown",
+
+	JSONEncodeError: "JSON Encoding Error",
+
+	BCCGetFromTableError:      "BCCGetFromTableError",
```

Comment:
I'm not sure whether we should use this error message scheme. I would prefer for them to be more human-readable, but I will leave it for you to decide